src/utils/d2c_messaging/src/d2c_messaging.c

```diff
@@ -526,12 +526,12 @@ bool ADUC_D2C_Message_SendAsync(
     {
         if (s_pendingMessageStore[type].completedCallback != NULL)
         {
-            Log_Debug("Replacing existing pending message. (t:%d)", type);
+            Log_Debug("Replacing existing pending message. (t:%d, s:%s)", type, s_pendingMessageStore[type].content);
             OnMessageProcessingCompleted(&s_pendingMessageStore[type], ADUC_D2C_Message_Status_Replaced);
         }
     }
 
-    Log_Debug("Queueing message (t:%d, c:0x%x", type, message);
+    Log_Debug("Queueing message (t:%d, c:0x%x, t:%s)", type, message, message);
```

Comment:
format for 3rd arg is `t:%s`
Did you mean to have value for 3rd arg be essentially `tostring(type)`, or should it be `s:%s` for the type's pending msg store content?